jobbergate-slurp/slurp/migrators/job_scripts.py

```diff
@@ -56,7 +56,24 @@ def migrate_job_scripts(nextgen_db, legacy_job_scripts, user_map, application_ma
         nextgen_jobscript_id = result.fetchone()["id"]
         job_scripts_map[job_script["id"]] = nextgen_jobscript_id
 
-        s3man.nextgen[nextgen_jobscript_id] = job_script["job_script_data_as_string"]
+        try:
+            job_script_files = s3man.nextgen.get_from_json(
+                job_script["job_script_data_as_string"],
+            )
+        except (ValueError, TypeError):
+            logger.error(
+                f"Error getting job script content {nextgen_jobscript_id=}: {job_script['job_script_data_as_string']}"
+            )
+        else:
+            if not job_script_files.main_file:
+                logger.warning(
+                    f"Empty job script content ({nextgen_jobscript_id=}; legacy_jobscript_id={job_script['id']})"
+                )
+                legacy_id_empty_jobscript.add(job_script["id"])
```

Comment:
We aren't skipping the empty job scripts?

Reply:
We are now skipping the empty job scripts on https://github.com/omnivector-solutions/jobbergate/pull/181